resources/logical_interconnect_group.rb

```diff
@@ -38,15 +40,57 @@ def load_lig(item)
         raise "#{up[:networkType]} #{network_name} not found" unless net.retrieve!
         up.add_network(net)
       end
-      uplink_info[:connections].each { |link| up.add_uplink(link[:bay], link[:port]) }
+      parsed_uplink_info[:connections].each { |link| up.add_uplink(link[:bay], link[:port]) }
       item.add_uplink_set(up)
     end
     item
   end
+
+  # like? function specific to the interconnectMapTemplate resource
+  def interconnect_map_like?(item, another_item)
```

Comment:
Yeah... It would probably be the better alternative, it would simplify a lot the work on the Chef resources.

Do you think we should remove these things and leave the comparison to the Ruby SDK, and if it's not working as we expect we should open an issue there? @jsmartt @hdiomede